### DISCUSSION TITLE
ultralytics platform yolov8-cls/yolov11-cls

### DISCUSSION THREAD

**Discussion by tcpipchip** [ultralytics/ultralytics#24132] [General]
Hi,

I am using  ultralytics platform to  train my monkey database class, using the YOLO8N–CLS, all going well, i can test the ultralytics predict to certify the correct train and after converted to my SG2002 hardware, all inferences goes well, near to 100%

When i used the YOLO11N-CLS with my same monkey database, i see something weird in my hardware inference, all with values to much far to 99%, then i back to to ultralytics platform and see the same on predict 

not detects “BARBIE” monkey

<img width="1600" height="860" alt="image" src="https://github.com/user-attachments/assets/bd98e3a7-11ba-40ea-9c60-26c54cd82103" />

and backing to the yolov8n-cls, all is good, its “BARBIE” monkey

<img width="1600" height="860" alt="image" src="https://github.com/user-attachments/assets/7e13024c-e185-4e8b-a25b-e2867cd1e275" />

Can you see that ?

Thanks

PS: sorry if duplicated question

**Comment by UltralyticsAssistant** [comment]
👋 Hello @tcpipchip, thank you for reporting this and for using the Ultralytics Platform 🚀 This is an automated response to help gather the right details, and an Ultralytics engineer will also assist you soon.

We recommend a quick visit to the [Docs](https://docs.ultralytics.com/) for new users, where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples and where many common questions may already be answered.

Since this appears to be a possible 🐛 model behavior issue between YOLO8-CLS and YOLO11-CLS on the same dataset, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) if possible. For this type of report, it would be especially helpful to include:
-Your Ultralytics Platform project or training settings used for both models
-The exact model variants tested
-Classification metrics, confusion matrix, and training logs for both runs
-A few example images for the affected class, including “BARBIE” monkey
-The exported model format used on your SG2002 hardware
-Any preprocessing or postprocessing differences between Platform prediction and hardware inference

If this is a custom training ❓ question, please also provide as much information as possible, including dataset image examples and training logs, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧 Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.

## Upgrade

Please upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to verify this is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments, with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled:

-**Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo26"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
-**Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
-**Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
-**Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

Thanks for including the screenshots 🙏 They’re helpful for showing the prediction difference between the two classification models, and with the additional details above, the team should be able to investigate more effectively.

**Comment by tcpipchip** [comment]
i can send you particulay thet monkey class database.

**Reply by glenn-jocher** [reply]
Thanks — we can’t review datasets privately, but if you share a small public sample and the exact train/export settings, plus the per-class metrics or confusion matrix and text prediction output for the same image, we can check it here; a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) is the fastest path, and for a quick sanity check I’d also try the same data with [Ultralytics YOLO26 classification](https://docs.ultralytics.com/tasks/classify/).

**Comment by tcpipchip** [comment]
https://drive.google.com/file/d/1JgfaV5m7JGyrfLMR0bUpZynCBvV81YYY/view?usp=sharing
44 class

**Comment by tcpipchip** [comment]
success  trainning on COLAB yolo11n-cls

**Reply by glenn-jocher** [reply]
Thanks for confirming `yolo11n-cls` works in Colab; that suggests an Ultralytics Platform/export-side issue or a class-order mismatch, so please verify the exported model keeps the same `model.names` order and open a new report in [GitHub Issues](https://github.com/ultralytics/ultralytics/issues) with the Platform run settings and a small sample per our [MRE guide](https://docs.ultralytics.com/help/minimum-reproducible-example/) so we can reproduce it.

**Comment by tcpipchip** [comment]
please, test with that google drive...and using the yolo11n default values on ultralytics platform

**Reply by glenn-jocher** [reply]
Thanks for reporting it — I can’t test from a Google Drive link here, but if `YOLO11n-cls` is consistently missing `BARBIE` in Ultralytics Platform `Predict`, please attach a small public sample for that class plus the exact training settings and validation/confusion-matrix results so the team can reproduce it; lower confidence can vary across models, but a repeatable wrong `top1` prediction is worth investigating, and our [Minimum Reproducible Example guide](https://docs.ultralytics.com/help/minimum-reproducible-example/) shows the best format.

**Comment by Y-T-G** [comment]
What's the accuracy you achieved with YOLO11 training? It seems like overfitting which can occur if your training dataset is small.

**Reply by tcpipchip** [reply]
> What's the accuracy you achieved with YOLO11 training? It seems like overfitting which can occur if your training dataset is small.

low, but get bigger on colab

**Comment by tcpipchip** [comment]
wow, i dont have more this info! Sorry!

**Reply by Y-T-G** [reply]
Can you post the link to your Platform projects for both YOLO11 and YOLO26?
